src/sentry/static/sentry/app/components/group/seenInfo.jsx

```diff
@@ -46,7 +46,7 @@ const SeenInfo = createReactClass({
   getTooltipTitle() {
     let {date, dateGlobal, environment, title} = this.props;
 
-    return componentToString(
+    return ReactDOMServer.renderToStaticMarkup(
```

Comment:
Let's keep `componentToString` here since we've updated its definition to use `renderToStaticMarkup`